src/command_modules/azure-cli-ml/azure/cli/command_modules/ml/env.py

```diff
@@ -428,19 +433,20 @@ def env_setup(status, name, kubernetes, context=CommandLineInterfaceContext()):
 
     env_verb = 'export' if context.os_is_linux() else 'set'
     env_statements = []
-    if kubernetes:
-        k8s_configured = create_action_with_prompt_if_defined(context, 'Kubernetes Cluster', OrderedDict([
-            ('Kubernetes Cluster Name', KubernetesOperations.get_cluster_name(context))
-        ]), setup_k8s, [context, root_name, resource_group, acr_login_server,
-                        acr_password, ssh_public_key, ssh_private_key_path])
-        if k8s_configured is True:
-            env_statements.append('{} AML_ACS_IS_K8S=True'.format(env_verb))
-    else:
-        create_action_with_prompt_if_defined(context, 'ACS', OrderedDict([
-            ('ACS Master URL', context.acs_master_url),
-            ('ACS Agent URL', context.acs_agent_url)]
-        ), az_create_acs, [root_name, resource_group, acr_login_server,
-                           context.acr_username, acr_password, ssh_public_key])
+    if not local_only:
+        if kubernetes:
+            k8s_configured = create_action_with_prompt_if_defined(context, 'Kubernetes Cluster', OrderedDict([
```

Comment:
refactor this to create prams on separate lines and then just pass them in to the method call and avoid all the line splitting